x-pack/plugins/ml/public/application/jobs/new_job/pages/components/common/json_editor_flyout/json_editor_flyout.tsx

```diff
@@ -42,18 +44,32 @@ interface Props {
   jobEditorMode: EDITOR_MODE;
   datafeedEditorMode: EDITOR_MODE;
 }
+
+const fetchSchemas = memoize(
+  async (jsonSchemaApi, path: string, method: string) =>
+    jsonSchemaApi.getSchemaDefinition({
+      path,
+      method,
+    }),
+  (jsonSchemaApi, path, method) => path + method
+);
+
 export const JsonEditorFlyout: FC<Props> = ({ isDisabled, jobEditorMode, datafeedEditorMode }) => {
   const { jobCreator, jobCreatorUpdate, jobCreatorUpdated } = useContext(JobCreatorContext);
   const { displayErrorToast } = useToastNotificationService();
   const [showJsonFlyout, setShowJsonFlyout] = useState(false);
   const [showChangedIndicesWarning, setShowChangedIndicesWarning] = useState(false);
 
+  const { jsonSchema: jsonSchemaApi } = useMlApiContext();
+
   const [jobConfigString, setJobConfigString] = useState(jobCreator.formattedJobJson);
   const [datafeedConfigString, setDatafeedConfigString] = useState(
     jobCreator.formattedDatafeedJson
   );
   const [saveable, setSaveable] = useState(false);
   const [tempCombinedJob, setTempCombinedJob] = useState<CombinedJob | null>(null);
+  const [jobSchema, setJobSchema] = useState<object>();
+  const [datafeedSchema, setDatefeedSchema] = useState<object>();
```

Comment:
Typo - should be `SetDatafeedSchema`

Reply:
Fixed in https://github.com/elastic/kibana/pull/146968/commits/4fb4d582b41222687a9e78f6ad30e20325055a6c